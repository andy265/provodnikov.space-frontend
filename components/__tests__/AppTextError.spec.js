const { createRenderer } = require('vue-server-renderer')
const { shallow } = require('vue-test-utils')
const AppTextError = require('~/components/AppTextError')

describe('AppTextError.vue', () => {
  it('has same HTML structure', () => {
    let wrapper = shallow(AppTextError, {
      slots: {
        default: '<span>some text</span>'
      }
    })
    let renderer = createRenderer()
    // eslint-disable-next-line
    renderer.renderToString(wrapper.vm, (err, str) => {
      expect(str).toMatchSnapshot()
    })
  })
})
