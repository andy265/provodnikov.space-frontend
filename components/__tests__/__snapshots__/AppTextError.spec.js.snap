// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AppTextError.vue has same HTML structure 1`] = `<p class="app-text-error"><span>some text</span></p>`;
